packages/gatsby/src/utils/page-data.ts

```diff
@@ -134,7 +135,7 @@ export async function flush(): Promise<void> {
 
   const pagesToWrite = pagePaths.values()
 
-  for (const pagePath of pagesToWrite) {
+  const flushQueue = fastq(async (pagePath, cb) => {
```

Comment:
```suggestion
  const flushQueue = fastq((pagePath, cb) => {
```
I don't think we need this (`fastq` doesn't seem to use promises)

Reply:
we do for the await

Reply:
oh, right